components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -2391,6 +2393,29 @@ private List<IdentityProviderProperty> filterIdentityProperties(IdentityProvider
                 }
             });
         }
+
+        identityProviderProperties.forEach(identityProviderProperty -> {
+            switch (identityProviderProperty.getName()) {
+                case IdPManagementConstants.FEDERATED_ASSOCIATION_ENABLED:
+                    federatedAssociationConfig
+                            .setEnabled(Boolean.parseBoolean(identityProviderProperty.getValue()));
+                    break;
+                case IdPManagementConstants.LOOKUP_ATTRIBUTES:
+                    String[] attributes = identityProviderProperty.getValue().split(",");
```

Comment:
Is there any chance for `identityProviderProperty.getValue()` to be null? If so, shall we add a null check here?

Reply:
Addressed with [3c20442](https://github.com/wso2/carbon-identity-framework/pull/5017/commits/3c20442145ac5549fb011ce7fae823d2970cfbe9)